Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginManager.swift

```diff
@@ -0,0 +1,113 @@
+    public init(accessGroup: String,
+                client: Client,
+                env: ClientConfiguration.Environment, // TODO: Currently used to decide language.
+                withMFA: MFAType? = nil,
+                loginHint: String? = nil,
+                extraScopeValues: Set<String> = [],
+                completion: @escaping LoginResultHandler) {
+        self.keychainSessionStorage = KeychainSessionStorage(service: Client.keychainServiceName, accessGroup: accessGroup)
+        self.client = client
+        
+        self.env = env
+        self.withMFA = withMFA
+        self.loginHint = loginHint
+        self.extraScopeValues = extraScopeValues
+        self.completion = completion
+    }
+    
+    @available(iOS 13.0, *)
+    public init(accessGroup: String,
+                client: Client,
+                env: ClientConfiguration.Environment, // TODO: Currently used to decide language.
+                withMFA: MFAType? = nil,
+                loginHint: String? = nil,
+                extraScopeValues: Set<String> = [],
+                withSSO: Bool = true,
+                completion: @escaping LoginResultHandler) {
+        self.keychainSessionStorage = KeychainSessionStorage(service: Client.keychainServiceName, accessGroup: accessGroup)
+        self.client = client
+        
+        self.env = env
+        self.withMFA = withMFA
+        self.loginHint = loginHint
+        self.extraScopeValues = extraScopeValues
+        self.completion = completion
+        self.withSSO = withSSO
+    }
+    
+    // MARK: -
+    
+    public func getSimplifiedLogin(completion: @escaping (Result<UIViewController, Error>) -> Void) throws {
+        let latestUserSession = self.keychainSessionStorage?.getAll()
+            .sorted { $0.updatedAt > $1.updatedAt }
+            .first
+        
+        guard let sLatestUserSession = latestUserSession else {
+            throw SimplifiedLoginError.noLoggedInSessionInSharedKeychain
+        }
+        
+        let user = User(client: client, tokens: sLatestUserSession.userTokens)
+        self.user = user
+        
+        user.userContextFromToken { result in
+            switch result {
+            case .success(let userContextResponse):
+                self.fetchProfile(user: user, userContext: userContextResponse, completion: completion)
+            case .failure(let error):
+                print("Some error happened \(error)")
+                completion(.failure(error))
+            }
+        }
+    }
+    
+    private func fetchProfile(user: User, userContext: UserContextFromTokenResponse, completion: @escaping (Result<UIViewController, Error>) -> Void) {
+        
+        user.fetchProfileData { result in
+            switch result {
+            case .success(_): // TODO: profileResponse and userContext need to be passed to factory when building SimplifiedLogin ViewController
```

Comment:
Is this comment still appropriate?

Reply:
Yes! data the `locale` and the `name` from `profileResponse` need to be passed into the factory.